config/config.go

```
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/davecgh/go-spew/spew"
+
+	"github.com/spf13/viper"
+)
+
+type config struct {
+	CSV struct {
+		Path string
+	}
+	Server struct {
+		Port string
+	}
+}
+
+var C config
+
+func ReadConfig() {
+	Config := &C
+
+	viper.SetConfigName("config")
+	viper.SetConfigType("yml")
+	path, _ := os.Getwd()
```

Never ignore errors.